Formula/k/kertish-dos.rb

```diff
@@ -1,11 +1,11 @@
-class KertishDfs < Formula
-  desc "Kertish FileSystem and Cluster Administration CLI"
-  homepage "https://github.com/freakmaxi/kertish-dfs"
-  url "https://github.com/freakmaxi/kertish-dfs/archive/refs/tags/v22.2.0147.tar.gz"
+class KertishDos < Formula
+  desc "Kertish Object Storage and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dos"
```

Comment:
can you cut a new release per the project rename?

Reply:
There will not be anything new soon (at least for 3 months) other than the renaming. Is it a kind of blocker for this PR for homebrew?

Reply:
nope, but it would be nice, otherwise, we dont need to regenerate the artifacts.

Reply:
actually, we still need to generate the bottle per the spec change.